src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -34,7 +34,7 @@ public class FileExpressionEvaluationTests : PowerFxTest
         public void InterpreterTestCase(ExpressionTestCase testCase)
         {
             // This is running against embedded resources, so if you're updating the .txt files,
-            // make sure they build is actually copying them over.
+            // make sure they build is actually copying them over.ab
```

Comment:
>ab [](http://example.com/codeflow?start=66&length=2)

typo?

Reply:
Oops, typo.  It's how I trigger the tests to rebuild after a .txt change without having to rebuild everything.